.github/workflows/ci-core-reusable.yml

```diff
@@ -99,13 +99,16 @@ jobs:
           ci_run git config --global --add safe.directory /usr/src/zksync/contracts
 
           ci_run zk
-          ci_run zk init
+          ci_run zk init --chain-id-hack
 
-      # `sleep 30` because we need to wait until server added all the tokens
+      # `sleep 60` because we need to wait until server added all the tokens
       - name: Run server
         run: |
-          ci_run zk server --uring --components api,tree,eth,state_keeper,housekeeper &>server.log &
-          ci_run sleep 30
+          ci_run zk server --uring --components api,tree,eth,state_keeper,housekeeper,commitment_generator &>server.log &
+          ci_run sleep 60
+
+      - name: Deploy legacy era contracts
```

Comment:
nit: 'upgrade-shared-bridge-era' doesn't sound like 'deploy legacy era contracts'..

We should either improve the name here or the zk argument name.